pkg/api/mlflow/common/helpers.go

```diff
@@ -1,6 +1,62 @@
 package common
 
+import (
+	"mime"
+	"path"
+	"strings"
+
+	"golang.org/x/exp/slices"
+)
+
+// textTypes used by GetContentType.
+var textTypes = []string{
+	"txt",
+	"log",
+	"err",
+	"cfg",
+	"conf",
+	"cnf",
+	"cf",
+	"ini",
+	"properties",
+	"prop",
+	"hocon",
+	"toml",
+	"yaml",
+	"yml",
+	"xml",
+	"json",
+	"js",
+	"py",
+	"py3",
+	"csv",
+	"tsv",
+	"md",
+	"rst",
+	"MLmodel",
+	"mlproject",
+}
+
 // GetPointer returns pointer for provided string.
 func GetPointer[T any](str T) *T {
 	return &str
 }
+
+// GetFilename returns the final bit of the path (the filename).
+func GetFilename(fullpath string) string {
+	_, filename := path.Split(fullpath)
+	return filename
+}
```

Comment:
why did you reinvent the wheel instead of using `filepath.Base`?

Reply:
The path at hand is a string so it needs to be parsed either way, right? This seems pretty straightforward.

Reply:
But why create a helper function `GetFilename` and its corresponding unit test when you can use an existing function from the standard library? This is the opposite of straightforward!

Reply:
changed to use filepath.Base at calling site